presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -29,7 +29,7 @@ Integer
 ``TINYINT``
 ^^^^^^^^^^^
 
-    A 8-bit signed two's complement integer with a minimum value of
+    An 8-bit signed two's complement integer with a minimum value of
```

Comment:
Can you either put this into a separate commit or raise a separate PR?

Reply:
OK, I reverted this change for now. I don't really care about it. I just made it to make sure the CI now behaved as expected.